data/src/main/java/com/mashup/data/source/remote/source/paging/KeyLinkPagingSource.kt

```diff
@@ -21,14 +22,16 @@ class KeyLinkPagingSource<T : Any>(
 
     override suspend fun load(params: LoadParams<Int>): LoadResult<Int, T> {
         val page = params.key ?: INITIAL_PAGE
+        val loadSize = params.loadSize
 
         return try {
-            val result = executor.invoke(page)
+            val result = executor.invoke(page, loadSize)
+            val isLastPage = page == result.paging?.totalPage
 
             LoadResult.Page(
-                data = result,
+                data = result.data,
                 prevKey = if (page == INITIAL_PAGE) null else page - 1,
-                nextKey = if (result.isEmpty()) null else page + 1
+                nextKey = if (isLastPage == true) null else page + 1
```

Comment:
```suggestion
                nextKey = if (isLastPage) null else page + 1
```
요렇게 바로 사용 가능할 것 같은뎅~?

Reply:
null 처리 때문에 그렇심다!